Code/src/megadesk.cpp

```diff
@@ -55,7 +60,8 @@
 uint16_t oldHeight = 0; // previously reported height
 #endif
 
-#define HYSTERESIS    137
+#define HYSTERESIS    137    // ignore movement requests < this distance
+#define MOVE_OFFSET   159    // amount to move when moving manually
```

Comment:
MOVE_OFFSET is chosen so that targetHeight is mostly accurate as soon as manual move button is released